### PR TITLE
zig zag order

### PR DESCRIPTION
**Pull request by NikitaKarbazin**
- [DEMO LINK](https://NikitaKarbazin.github.io/layout_snake/)
- [TEST REPORT LINK](https://NikitaKarbazin.github.io/layout_snake/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
```
Comment (Olexandr6) [inline src/style.css:5]: Always set fonts in the `html` selector. It's better approach

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+  .box-4 {
+    order: 5;
+  }
+  .box-5 {
+    order: 4;
+  }
+  .box-6 {
+    order: 3;
+  }
+}
```
Comment (Olexandr6) [inline src/style.css:140]: Omg, you don't need such a count of media queries in this task. Try to improve this, using only 2 media queries🙂

**Review by G-YaSha — CHANGES_REQUESTED** [review]
Your solution must be improved, check comments👇

**Review by G-YaSha — APPROVED** [review]
Looks much better🔥

**Pull request closed without merging**